models/reviews-models.js

```diff
@@ -63,3 +63,35 @@ exports.selectReviewId = (reviewId) => {
     return reviewCheck.rows;
   });
 };
+
+exports.updateReviewData = (reviewId, increment) => {
+  if (isNaN(Number(reviewId))) {
+    return Promise.reject({
+      status_code: 400,
+      message: `Invalid review ID provided`,
+    });
```

Comment:
PSQL error will be thrown if id is not a number please make change

Reply:
I'm a bit confused about this one - are you saying I should let PSQL do the error handling and remove checks like this?